example/index.ios.js

```diff
@@ -33,11 +33,12 @@ class logIn extends Component {
           style={styles.button}
           onPress={() => { 
             //Start Auth process
-            SpotifyModule.loggedIn((res) => {
+            SpotifyModule.loggedIn((res, accessToken) => {
               console.warn(res)
               if(!res) {
-                SpotifyModule.startAuthenticationFlow((error) => {
-                    if(!error){
+                SpotifyModule.startAuthenticationFlow((error, str) => {
+                  if(!error){
+                      console.log("New Access Token = "+str);
```

Comment:
I would change this to [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
`` `New Access Token ${str}` ``